src/views/modals/DeleteModal/DeleteModal.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
```

Comment:
Was this copied over from console? We should avoid reimplementing console functionality and instead open an issue to expose the delete modal.